pkgs/by-name/pw/pwalarmd/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, alsa-lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pwalarmd";
+  version = "0.1.0";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ alsa-lib ];
+
+  src = fetchFromGitHub {
+    owner = "amyipdev";
+    repo = pname;
```

Comment:
Not really sure about this. I have seen comments about preferring "pwalarmd" instead of variable name.

Reply:
Yeah it should be "pwalarmd"; instead of pname. See https://github.com/NixOS/nixpkgs/issues/277994